requirements.txt

```diff
@@ -0,0 +1,7 @@
+opencv-python-headless==4.4.0.44
+matplotlib==3.3.2
+numpy==1.19.1
+scipy==1.5.2
+tensorflow==1.15.3
```

Comment:
What are we using tensorflow for?

Reply:
Tensorboard logging is what it is used for